lib/events.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const P = require('./promise')
+
+const ACTIVITY_EVENTS = new Set([
+  'account.confirmed',
+  'account.created',
+  'account.deleted',
+  'account.keyfetch',
+  'account.login',
+  'account.reset',
+  'account.signed',
+  'account.verified',
+  'device.created',
+  'device.deleted',
+  'device.updated'
+])
+
+// We plan to emit a vast number of flow events to cover all
+// kinds of success and error steps of the sign-in/up journey.
+// It's far easier to define the events that *aren't* a flow
+// event than it is to define those that are.
+const NOT_FLOW_EVENTS = new Set([
+  'account.deleted',
+  'account.reset',
+  'device.created',
+  'device.deleted',
+  'device.updated'
+])
+
+const NOT_FLOW_EVENTS_IF_SERVICE_IS_CONTENT_SERVER = new Set([
```

Comment:
Is there a situation where we would ignore from other services? If so, maybe make it `IGNORE_FLOW_EVENT_FROM_SERVICE`, where each object has name and service?